rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -549,7 +557,26 @@ EOF
     sed -i 's/^\(# This is too little for Rudder, as there are some big entries.*\)/\1\nsockbuf_max_incoming_auth 1000000000/' /opt/rudder/etc/openldap/slapd.conf
     echo " Done"
   fi
-}
+
+  # 6.1.0~beta3:
+  # Add indexes cn,softwareVersion,directiveId in slapd.conf
+  SLAPD_CONF="/opt/rudder/etc/openldap/slapd.conf"
+  NEED_REINDEX=false
+  for i in "directiveId" "softwareVersion" "cn"; do
+    line="index\s*${i}\s*eq"
+    grep -E "${line}" ${SLAPD_CONF} > /dev/null 2>&1 || (
+      echo "Adding LDAP index on attribute: ${i}"
+      sed -i "/index\s*objectClass\s*eq/a index\t${i}\teq" ${SLAPD_CONF}
+      NEED_REINDEX=true
+    )
+  done
+  if "${NEED_REINDEX}"; then
```

Comment:
```suggestion
  if ${NEED_REINDEX}; then
```

Reply:
almost everything is a text in so we need comparison in tests : if [ "${NEED_REINDEX}" = "true" ]; then